2024-02-26 10:09:19,529 INFO    MainThread:27161 [wandb_setup.py:_flush():76] Current SDK version is 0.16.0
2024-02-26 10:09:19,530 INFO    MainThread:27161 [wandb_setup.py:_flush():76] Configure stats pid to 27161
2024-02-26 10:09:19,530 INFO    MainThread:27161 [wandb_setup.py:_flush():76] Loading settings from /home/adriano/.config/wandb/settings
2024-02-26 10:09:19,530 INFO    MainThread:27161 [wandb_setup.py:_flush():76] Loading settings from /home/adriano/Desktop/tiago_ws/src/elsa_tiago_fl/wandb/settings
2024-02-26 10:09:19,530 INFO    MainThread:27161 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-26 10:09:19,530 INFO    MainThread:27161 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-26 10:09:19,530 WARNING MainThread:27161 [wandb_setup.py:_flush():76] Could not save program above cwd: /home/adriano/.pyenv/versions/elsa_env/lib/python3.10/site-packages/ray/_private/workers/default_worker.py
2024-02-26 10:09:19,530 INFO    MainThread:27161 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program_abspath': '/home/adriano/.pyenv/versions/elsa_env/lib/python3.10/site-packages/ray/_private/workers/default_worker.py', 'program': '/home/adriano/.pyenv/versions/elsa_env/lib/python3.10/site-packages/ray/_private/workers/default_worker.py'}
2024-02-26 10:09:19,530 INFO    MainThread:27161 [wandb_init.py:_log_setup():524] Logging user logs to save/wandb/run-20240226_100919-r93a90yc/logs/debug.log
2024-02-26 10:09:19,530 INFO    MainThread:27161 [wandb_init.py:_log_setup():525] Logging internal logs to save/wandb/run-20240226_100919-r93a90yc/logs/debug-internal.log
2024-02-26 10:09:19,530 INFO    MainThread:27161 [wandb_init.py:init():564] calling init triggers
2024-02-26 10:09:19,530 INFO    MainThread:27161 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {'Model': 'dqn_new', 'Env': 'TiagoSimpleEnv-v0', 'Batch size': 64, 'lr': '1e-4', 'seed': 42, 'FL Flower': True, 'Sample Clients': 1, 'Total Clients': 1, 'FL Rounds': 5, 'Iterations per FL Round': 1}
2024-02-26 10:09:19,530 INFO    MainThread:27161 [wandb_init.py:init():614] starting backend
2024-02-26 10:09:19,530 INFO    MainThread:27161 [wandb_init.py:init():618] setting up manager
2024-02-26 10:09:19,533 INFO    MainThread:27161 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-02-26 10:09:19,535 INFO    MainThread:27161 [wandb_init.py:init():624] backend started and connected
2024-02-26 10:09:19,542 INFO    MainThread:27161 [wandb_init.py:init():716] updated telemetry
2024-02-26 10:09:19,550 INFO    MainThread:27161 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2024-02-26 10:09:19,985 INFO    MainThread:27161 [wandb_run.py:_on_init():2254] communicating current version
2024-02-26 10:09:20,034 INFO    MainThread:27161 [wandb_run.py:_on_init():2263] got version response upgrade_message: "wandb version 0.16.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-02-26 10:09:20,034 INFO    MainThread:27161 [wandb_init.py:init():800] starting run threads in backend
2024-02-26 10:09:20,206 INFO    MainThread:27161 [wandb_run.py:_console_start():2233] atexit reg
2024-02-26 10:09:20,207 INFO    MainThread:27161 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-02-26 10:09:20,209 INFO    MainThread:27161 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-02-26 10:09:20,210 INFO    MainThread:27161 [wandb_run.py:_redirect():2178] Redirects installed.
2024-02-26 10:09:20,213 INFO    MainThread:27161 [wandb_init.py:init():841] run started, returning control to user process
2024-02-26 10:09:20,218 INFO    MainThread:27161 [wandb_run.py:_config_callback():1342] config_cb None None {'Model Params': 0, 'Model Trainable Params': 0}
2024-02-26 10:12:54,199 WARNING MsgRouterThr:27161 [router.py:message_loop():77] message_loop has been closed
